tests/Fixtures/TestBundle/Resources/config/api_resources_orm/resources.xml

```diff
@@ -15,7 +15,7 @@
         </uriVariables>
 
         <operations>
-            <operation class="ApiPlatform\Metadata\GetCollection" name="custom_operation_name"/>
```

Comment:
It is needed since the IRI converter uses the operation name:
https://github.com/api-platform/core/blob/f1ecc30a38e50536a2a65ae85ef23eb6dc095af3/src/Symfony/Routing/IriConverter.php#L157
In the previous version of the `ExtractorResourceMetadataCollectionFactory`, the name was overridden. The `AttributesResourceMetadataCollectionFactory` does not do that.
Does it mean we cannot customize the operation name @soyuka?

Reply:
yes we can change it, it should not be overriden

Reply:
but usually the name of the operation is used to define the one inside the router therefore this should work as welll

Reply:
Fixed!